slides/01-introduction.qmd

```diff
@@ -234,45 +251,36 @@ abalone <- read_csv("abalone.csv")
 
 ```{python}
 import pandas as pd
-abalone = pd.read_csv("abalone.csv")  
+pd.read_parquet('./data/inspections.parquet', engine = 'pyarrow')
```

Comment:
Is this right @isabelizimm?

Reply:
Yep! I'll just make sure that pyarrow is in the appropriate requirements files